google/self_link_helpers.go

```diff
@@ -85,3 +85,18 @@ func NameFromSelfLinkStateFunc(v interface{}) string {
 func StoreResourceName(resourceLink interface{}) string {
 	return GetResourceNameFromSelfLink(resourceLink.(string))
 }
+
+// GetZoneFromSelfLink will attempt to parse the zone if it's in the referenced self link
+//
+// If there is no zone present or the link is malformed it will return an empty string
+func GetZoneFromSelfLink(link string) string {
```

Comment:
I flip flopped on this one a bunch. I was thinking that the `.Get` methods all just return default values if they haven't been set so was aiming for that behavior. But I suppose since `getZone` returns an error it would be in keeping with that.